src/modules/messages/proxyPluralMessage.ts

```diff
@@ -37,16 +43,26 @@ export const proxyPluralMessage = async (
       allowedMentions: {
         parse: [],
       },
+      embeds: [],
     };
 
     if (message.type === MessageType.Reply && message?.reference?.messageId) {
       const originalMsg = await message.channel.messages.fetch(
         message.reference.messageId
       );
-      originalMsg.reply(content);
-    } else {
-      await webhook.send(content);
+
+      content.content =
+        (message.mentions.users.size ? `<@${originalMsg.author.id}>, ` : "") +
+        `${content.content}`;
+      content.embeds = [
+        new EmbedBuilder().setDescription(originalMsg.content).setAuthor({
+          name: originalMsg.author.username,
+          iconURL: originalMsg.author.displayAvatarURL(),
+          url: originalMsg.url,
+        }),
+      ];
```

Comment:
```suggestion
      content.embeds?.push(
        new EmbedBuilder().setDescription(originalMsg.content).setAuthor({
          name: originalMsg.author.username,
          iconURL: originalMsg.author.displayAvatarURL(),
          url: originalMsg.url,
        })
      );
```
Since the array exists already, we can push to it instead of reassigning with a new array.